commit 82c17cd232a016f0c552f38453774abfb0bf17fe
Author: James Cloos <cloos@jhcloos.com>
Date:   Wed Aug 20 10:32:48 2008 -0400

    xaw8 is gone, use xaw7

diff --git a/configure.ac b/configure.ac
index e34e9fe..dafd9f8 100644
--- a/configure.ac
+++ b/configure.ac
@@ -31,7 +31,7 @@ AM_CONFIG_HEADER(config.h)
 AC_PROG_CC
 AC_PROG_INSTALL
 
-XAW_CHECK_XPRINT_SUPPORT(XAW)
+PKG_CHECK_MODULES(XAW, xaw7)
 
 # Checks for pkg-config packages
 PKG_CHECK_MODULES(EDITRES, x11 xt xmu)
