--- editres-1.0.7/geometry.c.orig	2018-03-15 20:38:33.578823819 +0100
+++ editres-1.0.7/geometry.c	2018-03-15 20:45:03.065486046 +0100
@@ -63,7 +63,7 @@
     Window win;
     int x, y;			/* location of event in root coordinates. */
 
-    snprintf(msg, sizeof(msg), res_labels[14]);
+    snprintf(msg, sizeof(msg), "%s", res_labels[14]);
 
     SetMessage(global_screen_data.info_label, msg);
 
@@ -103,7 +103,7 @@
 		    find_event->widgets.num_widgets);
 
     if (node == NULL) {
-	snprintf(msg, sizeof(msg), res_labels[13]);
+	snprintf(msg, sizeof(msg), "%s", res_labels[13]);
 	SetMessage(global_screen_data.info_label, msg);
 	return;
     }
--- editres-1.0.7/svpopup.c.orig	2018-03-15 20:38:33.578823819 +0100
+++ editres-1.0.7/svpopup.c	2018-03-15 20:45:25.268819111 +0100
@@ -127,7 +127,7 @@
 	old = global_screen_data.res_text;
 	break;
     default:
-	snprintf(msg, sizeof(msg), res_labels[22]);
+	snprintf(msg, sizeof(msg), "%s", res_labels[22]);
 	SetMessage(global_screen_data.info_label, msg);
 	return;
     }
--- editres-1.0.7/utils.c.orig	2018-03-15 20:38:33.578823819 +0100
+++ editres-1.0.7/utils.c	2018-03-15 20:45:53.745485464 +0100
@@ -518,7 +518,7 @@
 			get_event->info[i].widgets.num_widgets);
 
 	if (node == NULL) {
-	    snprintf(buf, sizeof(buf), res_labels[16]);
+	    snprintf(buf, sizeof(buf), "%s", res_labels[16]);
 	    AddString(&errors, buf);
 	    continue;
 	}
