--- editres-1.0.6/geometry.c.orig	2013-01-13 01:05:30.000000000 +0100
+++ editres-1.0.6/geometry.c	2013-01-15 19:21:30.973960183 +0100
@@ -63,7 +63,7 @@ _FindWidget(Widget w)
     Window win;
     int x, y;			/* location of event in root coordinates. */
 
-    sprintf(msg, res_labels[14]);
+    snprintf(msg, BUFSIZ, "%s", res_labels[14]);
 
     SetMessage(global_screen_data.info_label, msg);
 
@@ -103,7 +103,7 @@ DisplayChild(Event *event)
 		    find_event->widgets.num_widgets);
 
     if (node == NULL) {
-	sprintf(msg, res_labels[13]);
+	snprintf(msg, BUFSIZ, "%s", res_labels[13]);
 	SetMessage(global_screen_data.info_label, msg);
 	return;	
     }
--- editres-1.0.6/svpopup.c.orig	2013-01-13 01:05:30.000000000 +0100
+++ editres-1.0.6/svpopup.c	2013-01-15 19:24:45.332921442 +0100
@@ -127,7 +127,7 @@ ModifySVEntry(Widget w, XEvent *event, S
 	old = global_screen_data.res_text;
 	break;
     default:
-	sprintf(msg, res_labels[22]);
+	snprintf(msg, BUFSIZ, "%s", res_labels[22]);
 	SetMessage(global_screen_data.info_label, msg);
 	return;
     }
--- editres-1.0.6/utils.c.orig	2013-01-13 01:05:30.000000000 +0100
+++ editres-1.0.6/utils.c	2013-01-15 19:25:25.208093290 +0100
@@ -518,7 +518,7 @@ HandleGetResources(Event *event)
 			get_event->info[i].widgets.num_widgets);
 
 	if (node == NULL) {
-	    sprintf(buf, res_labels[16]);
+	    snprintf(buf, BUFSIZ, "%s", res_labels[16]);
 	    AddString(&errors, buf); 
 	    continue;	
 	}
